timeline/urls.py

```diff
@@ -19,6 +19,11 @@
         view=views.SiteTimelineView.as_view(),
         name='list_site',
     ),
+    url(
+        regex=r'^admin_site$',
```

Comment:
In URLs we should also follow conventions closely, but I haven't really documented these properly so my bad :)

I would recommend changing this into `^site/all$`. We generally want to avoid underscores in URL slugs and the string "admin" can have some funny consequences (I've learned this the hard way..)

Reply:
- [x] Changed